modules/friendlytag.js

```diff
@@ -167,7 +219,7 @@ Twinkle.tag.updateSortOrder = function(e) {
 			checkbox.checked = true;
 		}
 		switch (tag) {
-			case "cleanup":
+			case "Cleanup":
```

Comment:
I think I did that to align all labels with the actual template names. Of course it's possible to make first char lowercase.  But the CSS classes on box templates are of form `box-Template_name` where the template name is ucfirst. So if we decide to go all lcfirst, we'll have to change L189 suitably too,

Reply:
Actually, thinking about it a bit, won't this make all previous tags ineligible?  That is, since Twinkle has been (largely) using lowercased tags, won't the check below via `linkshere` fail to remove a tag since now it's looking for uppercase tags?

Reply:
No it's looking for both ucfirst and lcfirst tags. But I remain completely open to idea of making them all lcfirst, since I guess people are used to that.